docs/how-tos/adding-events-to-event-bus.rst

```diff
@@ -21,21 +21,22 @@ In the producing/host application, include ``openedx_events`` in ``INSTALLED_APP
 
    # .. setting_name: EVENT_BUS_PRODUCER_CONFIG
    # .. setting_default: {}
-   # .. setting_description: Dictionary of event_types mapped to lists of dictionaries containing topic related configuration.
-   #    Each topic configuration dictionary contains
-   #    * a topic/stream name called `topic` where the event will be pushed to.
-   #    * a flag called `enabled` denoting whether the event will be published to the topic.
-   #    * `event_key_field` which is a period-delimited string path to event data field to use as event key.
+   # .. setting_description: Dictionary of event_types to dictionaries for topic related configuration.
+   #    Each topic configuration dictionary uses the topic as a key and contains a flag called `enabled`
+   #    denoting whether the event will be and `event_key_field` which is a period-delimited string path
+   #    to event data field to use as event key.
```

Comment:
I like how clear was the previous structure, can we consider something like this? I'm not sure how well it renders though.
```suggestion
   # .. setting_description: Dictionary of event_types to dictionaries for topic-related configuration.
   #    Each topic configuration dictionary uses the topic as a key and contains:
   #    * A flag called `enabled` denoting whether the event will be published.
   #    * The `event_key_field` which is a period-delimited string path to event data field to use as event key.
```